caampr-backend/src/main/java/com/jelistan/caampr/lambda/model/Link.java

```diff
@@ -1,12 +1,16 @@
 package com.jelistan.caampr.lambda.model;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Builder
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class Link {
-    private final int id;
-    private final String url;
-    private final LinkTypes type;
+    private int id;
+    private String url;
+    private LinkTypes type;
```

Comment:
note that this is probably gonna fail without the proper enum annotation. can be fixed in another commit

Reply:
todo, in issue 7